contracts/validator-manager/tests/PoSValidatorManagerTests.t.sol

```diff
@@ -617,6 +657,13 @@ abstract contract PoSValidatorManagerTest is ValidatorManagerTest {
         assertEq(v3, 1e27);
     }
 
+    function testPoSValidatorManagerStorageSlot() public view {
```

Comment:
was the change to storage locations just to make these tests possible?

Reply:
yup